examples/example_caching.py

```diff
@@ -98,7 +98,7 @@ def objective(individual):
 # Finally, we call the `evolve` method to perform the evolutionary search.
 
 
-cgp.evolve(pop, objective, ea, **params["evolve_params"], print_progress=True)
+cgp.evolve(objective, pop, ea, **params["evolve_params"], print_progress=True)
```

Comment:
I think we should be consistent in calling it `objective` or `obj` in all examples (but maybe this can be a separate maintenance PR)

Reply:
see separate issue #336